templates/README.md

```diff
@@ -54,6 +54,8 @@ This repository has a script that does a basic check if a GitHub repository/orga
 
 You run it like this: `jbang policypanda@commonhaus/foundation hibernate` where `hibernate` is the GitHub organization name.
 
+NOTE: The script requires GitHub credentials in the form of either environment variables (see [help here](https://github-api.kohsuke.org/#Environmental_variables), or [alternate link](https://web.archive.org/web/20250108173712/https://github-api.kohsuke.org/#Environmental_variables)) or a `~/.github` properties file (see [help here](https://github-api.kohsuke.org/#Property_file), or [alternate link](https://web.archive.org/web/20250108173712/https://github-api.kohsuke.org/#Property_file)).
+
```

Comment:
Wouldn't it be better to show how it's done? Besides linking for additional info.

Reply:
Maybe, but it's complicated. There are several options: login/password, oauth token, JWT token, ...
At least here the info is reachable, that's a start.

Reply:
Yea. I had to go look at those docs to get things set up, so the pointer to them is useful.

I was chewing on an alternative: add it to the admin bot. If you add the admin bot to the organization, it can look at everything and spit out / update a report (into foundation-internal, e.g.)